dijkstra/dijkstra.c

```diff
@@ -16,7 +16,7 @@
 int minimum_distance(const int min_distances[], const bool shortest_paths[]) {
 	int i;
 	int minimum = INFINITY, index;
-	for (int i = 0; i < VERTICES; ++i) {
+	for (i = 0; i < VERTICES; ++i) {
```

Comment:
I have missed that one while typing but it didn't show error locally or even on ideone, I'm also confused about how this happened logically it should show out error but it didn't.